pkg/api/http_server.go

```diff
@@ -76,6 +77,7 @@ type HTTPServer struct {
 	Bus                       bus.Bus
 	RenderService             rendering.Service
 	Cfg                       *setting.Cfg
+	Features                  *featuremgmt.FeatureManager
```

Comment:
Do we need FeatureManager here or could we use FeatureToggles?

Reply:
there is an [HTTP endpoint ](https://github.com/grafana/grafana/pull/44443/files#diff-67a6e36014be865053c43d38b2091bd2d3f17a91b117d97c0f5244cd7d71127dR464) that can list details and should eventually support changing state (ie, writing to DB or temporary change etc)

Perhaps FeatureManager could extend FeatureToggles so we do not need: `hs.Features.Toggles().IsLivePipelineEnabled()`


If we drop the `IsXXXEnabled()` helpers, this would be much easier, but the checks become a bit more verbose (maybe better/ok?):
```
s.features.IsEnabled(featuremgmt.FlagLivePipeline)
```

Reply:
yeah dropping the helper functions might make sense, we're already importing featuremgmt for the struct defs so that would make things cleaner and also allow us to accept a FeatureToggles interface that could be fulfilled by FeatureMgmt, and we'd still have static checking to make sure flags are valid.

Reply:
@DanCech just pushed a big refactor that uses a constant for everything